apps/web/src/pages/TrainingOpportunities/CreateTrainingOpportunityPage.tsx

```diff
@@ -201,7 +192,7 @@ const CreateTrainingOpportunityPage = () => {
     <>
       <SEO title={pageTitle} />
       <Hero title={pageTitle} crumbs={navigationCrumbs} overlap centered>
-        <div data-h2-margin-bottom="base(x3)">
+        <div className="mb-18">
```

Comment:
Not necessary but on other pages, the wrapping div was removed and the margin was moved to the card.

https://github.com/GCTC-NTGC/gc-digital-talent/blob/5a1b15734a5fb928b9275cfc2daa464d7d696f4f/apps/web/src/pages/WorkStreams/CreateWorkStreamPage.tsx#L145